contents/handbook/growth/marketing/newsletter.md

```diff
@@ -4,22 +4,24 @@ sidebar: Handbook
 showTitle: true
 ---
 
-We do not currently do email marketing, beyond product announcements via our newsletter, the PostHog Array. This is sent via Mailchimp. In the future, we will probably want to grow this channel. The Growth team use email for onboarding and other email flows, but this page covers marketing only. 
+We publish two newsletters:
 
-Speak to Charles, Yakko, or Charles for more information. 
+1. PostHog Array, a product announcement email published every four weeks
+2. PostHog News (working title), a marketing and content email published every two weeks
+
+Both emails are sent via Mailchimp. PostHog News (working title) is a work-in-progress newsletter which we plan to increase to a weekly schedule once we're confident we can produce it to a high standard at that higher frequency. PostHog News is primarily aimed at existing users as a means to improving engagement with our content and aid product discovery through sharing tutorials, tips and ideas, not user acquisition.
 
 ## Lists
 
 We currently grow our general email list from three sources:
+
 - Newsletter signups from the website
 - PostHog users who opt-in
 - Event attendees
```

Comment:
Are we actually doing all of these? From what I can tell, there is no opt-in procedure during account sign-up and I'd be surprised if we actually captured emails from Eventbrite from the recent event.